nodejs/week1/meal_app/src/backend/routes/cheap-meals.js

```diff
@@ -0,0 +1,21 @@
+const express = require("express");
+const app = express();
+
+const mealsJson = require(__dirname + '/../data/meals.json');
+const reviewsJson = require(__dirname + '/../data/reviews.json');
+
+const cheapMealsArray = mealsJson.filter(cheapMeals => cheapMeals.price < 50)
```

Comment:
A small thing - your naming is misleading here. Inside of `filter` function, you go through `all` meals, so:
1. As meals are not filtered at the moment the function runs, they are not `cheapMeals` yet. They rather all the meals you have.
2. Inside of `filter` function, you are going through every meal, so it's not plural.

I would call it just `meal`.

The same mistake repeats for some routes as well.

Reply:
Thank you Vlad for the feedback :)  I have implemented necessary changes. Waiting for your approval.
regards
krithi